swagger/control/swagger.yaml

```diff
@@ -0,0 +1,152 @@
+swagger: '2.0'
+info:
+  title: Mailgun Relayery Control API
+  description: |
+    API for managing the authentication tokens and channel settings of remote instances.
+  version: "1.0.0"
+host: mailgun.relayery.control.parquery.com
```

Comment:
I think host is not mandatory -- I'd just remove it; there is no need to expose parquery-internal architecture to the outside world.